common/flavorization/wc_title_holders.txt

```diff
@@ -162,149 +152,2026 @@ warlord_female = {
+	type = title
+	tier = barony
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+guardian_male_dryad_group_count = {
+    type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+guardian_female_dryad_group_count = {
+    type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+grove_dryad_group_county = {
+	type = title
+	tier = county
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+lord_male_dryad_group_duke = {
+    type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+lady_female_dryad_group_duke = {
+    type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+grand_grove_dryad_group_duchy = {
+	type = title
+	tier = duchy
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+high_lord_male_dryad_group_king = {
+    type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+high_lady_female_dryad_group_king = {
+    type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+domain_dryad_group_kingdom = {
+	type = title
+	tier = kingdom
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+king_male_dryad_group_emperor = {
+    type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+queen_female_dryad_group_emperor = {
+    type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 401
+	top_liege = no
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+kingdom_dryad_group_empire = {
+	type = title
+	tier = empire
+	priority = 401
+	culture_groups = { dryad_group }
+	religions = { druidism_group shamanism_group }
+}
+
+### Tol'vir ###
+# Feudal - Normal
+governor_male_tolvir_count = {
+    type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	cultures = { tolvir }
+}
+governor_female_tolvir_count = {
```

Comment:
I remember we had a conversation about governors.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/444#discussion_r653695949
It's much better to find some noble aristocratic title.

Reply:
I don't agree, the governors in Egypt and China were overwhelmingly In addition, these are titles inspired by that suddenly.

Reply:
> I don't agree, the governors in Egypt and China were overwhelmingly In addition, these are titles inspired by that suddenly.

I would rather find historical titles than use English translation in this case.

Reply:
Nomes are historically names. I can't find for others titles.

Reply:
Done in the last commits